cli/validation/converter/iam_test.go

```diff
@@ -0,0 +1,330 @@
+package converter
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/validation/cai"
+)
+
+func TestGetIamCaiObject(t *testing.T) {
```

Comment:
This is fine for now, but I'd like to avoid handwritten tests for each resource. Instead we should have cases similar to what we do for TFV.

Reply:
How do we do it for TFV?

Reply:
https://github.com/GoogleCloudPlatform/terraform-validator/blob/master/test/cli_test.go#L52